_index.md

```diff
@@ -43,7 +43,7 @@ The table lists known hardware cryptographic devices and their support, or non-s
 | Century Longmai PKI Token      | :x::clock1:        | CMS/PKCS#7            |                                                                                                                                                                                                                                                                                                                                                                                                                                          | Claimed roadmap item                                                                                                  |
 | TrustSec SLCOS - Bio/PKI token | :x:                |                       |                                                                                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                                       |
 | **Other Devices**              |
-| Apple iOS                      | :heavy_check_mark: | CBOR/WebAuthn         | https://developer.apple.com/documentation/devicecheck<br>https://developer.apple.com/documentation/devicecheck/dcappattestservice/3573911-attestkey<br>https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server                                                                                                                                                                                 | Not for the purpose of key attestation as defined here, perhaps it can be used for additional purposes than intended? |
+| Apple iOS                      | :heavy_check_mark: | X.509/ACME/CBOR/WebAuthn         | Managed Device Attestation (iOS 16 and later):<br/>https://support.apple.com/en-gb/guide/deployment/dep28afbde6a/web<br/>For Apps:<br/>https://developer.apple.com/documentation/devicecheck<br/>https://developer.apple.com/documentation/devicecheck/dcappattestservice/3573911-attestkey<br/>https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server                                                                                                                                                                                 | Managed Device Attestation is used for key attestation. App attestation does not seem to be for the purpose of key attestation as defined here. |
```

Comment:
Hi @primetomas, HTML is not allowed in our website markdown settings.

Reply:
that was already present since before, I see on the web page it was stripped now. Any other suggestions how to format text and multiple links so it is readable?

Reply:
Maybe we don't need to show the full link?
```md
[DeviceCheck (developer.apple.com)](https://developer.apple.com/documentation/devicecheck<br/>https://developer.apple.com/documentation/devicecheck/dcappattestservice/3573911-attestkey), [link 2](...) 
```